pkg/dbutils/dbutils.go

```diff
@@ -80,3 +86,83 @@ func QuoteLiteral(str string) string {
 		return `'` + res + `'`
 	}
 }
+
+func InitPublication(conn *pgx.Conn, publicationName string) error {
+	rows, err := conn.Query("select 1 from pg_publication where pubname = $1;", publicationName)
+	if err != nil {
+		return fmt.Errorf("could not execute query: %v", err)
+	}
+
```

Comment:
defer rows.Close() (noticed we are not doing it if publication creation errors out, but we still should.

Reply:
we do close it(https://github.com/ikitiki/logical_backup/pull/35/files#diff-87ca3ef81cadbba95ef6810c5ae9051cR97 and https://github.com/ikitiki/logical_backup/pull/35/files#diff-87ca3ef81cadbba95ef6810c5ae9051cR100)

Reply:
ah, right. But doing it in a defer avoids duplicating rows.Close() all over the function.